### ISSUE TITLE
Odd filenaming in tutorial/intro.xhtml

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11544]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| spiv reported|
|-|-|
|Trac ID|trac#637|
|Type|enhancement|
|Created|2004-06-18 06:06:40Z|
|Branch|https://github.com/twisted/twisted/tree/odd-filenaming-637|

Attachments:

* [odd-filenaming-637.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/88a/88a9d5a83b2b7e4bc74200cc205858df88a90f44/4b523fc35532ca0a3ce265215bd376390e653c9a.patch) (3016 bytes) - added by thijs on 2008-07-12 23:14:45Z - patch against r24262

<details><summary>Searchable metadata</summary>

```
trac-id__637 637
type__enhancement enhancement
reporter__spiv spiv
priority__high high
milestone__ 
branch__branches_odd_filenaming_637 branches/odd-filenaming-637
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1087538800000000 1087538800000000
changetime__1217430881000000 1217430881000000
version__ 
owner__ 
cc__spiv cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
It says, near the bottom: "After saving the next example (finger11.py) as
finger.tac ..."

How about we simply call the example finger11.tac?  It would make browsing the
tutorial for examples of tac files easier, aside from being less gratuitously
confusing :)
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
Suspect the solutions to 532 and 626 will supersede this, but leaving it open
for now.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement set owner to edsuom|
|-|-|

Reassigning doc bugs to edsuom.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Replying to [hypatia](#note_2):
> Suspect the solutions to 532 and 626 will supersede this, but leaving it open
> for now.

Tickets [[#11338](https://github.com/twisted/twisted/issues/11338)1513](https://github.com/twisted/twisted/issues/11513) and [#10471](https://github.com/twisted/twisted/issues/10471) are still open but I think we can work on this one before finishing the others. I will make a patch that supplies the suggested solution in comment #1.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Most (but not all) of these files seem actually to be tac files.  It'd be good to rename the rest, instead of just having `finger11.tac` sticking out, different from everything else, confusing people.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_8"></a>|@thijstriemstra commented|
|-|-|

(In [24442]) Branching to 'odd-filenaming-637'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_9"></a>|@thijstriemstra removed owner|
|-|-|

Replying to [exarkun](#note_7):
> Most (but not all) of these files seem actually to be tac files.  It'd be good to rename the rest, instead of just having `finger11.tac` sticking out, different from everything else, confusing people.

I renamed the tac services in r24444. Putting the branch up for review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_10"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_11"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24451]) Merge odd-filenaming-637: Fix odd filenaming in finger tutorial.

Author: thijs
Reviewer: exarkun
Fixes: [#11544](https://github.com/twisted/twisted/issues/11544)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
